RELEASE.txt

```diff
@@ -1,13 +0,0 @@
-8/17/2018
```

Comment:
I personally think it is a good idea to, at least, keep this file too. This format is much easier to read if you're working with a terminal or other non-web browser based application